utils/blog.ts

```diff
@@ -16,15 +16,18 @@ const LINK_DOMAINS: Partial<{ [name in TIcon]: string }> = {
 
 export const addLinkPrefix = (linkObject: ILink): string => {
   const { link, type } = linkObject;
-  if (link.startsWith('t.me')) return `https://${link}`;
+  if (link.startsWith('t.me')) return `https://${link.replace('@', '')}`;
```

Comment:
it will remove @ from all types of link
it will remobe @ from facebook links, linkedin links and from other links. But is should not